src/libraries/System.IO.Pipelines/ref/System.IO.Pipelines.cs

```diff
@@ -57,6 +57,8 @@ public abstract partial class PipeReader
         public System.Threading.Tasks.ValueTask<System.IO.Pipelines.ReadResult> ReadAtLeastAsync(int minimumSize, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         protected virtual System.Threading.Tasks.ValueTask<System.IO.Pipelines.ReadResult> ReadAtLeastAsyncCore(int minimumSize, System.Threading.CancellationToken cancellationToken) { throw null; }
         public abstract bool TryRead(out System.IO.Pipelines.ReadResult result);
+        protected internal static Func<Stream, ReadOnlyMemory<byte>, System.Threading.CancellationToken, System.Threading.Tasks.ValueTask<FlushResult>> WriteAsyncFunc() { throw null; }
```

Comment:
You shouldn't need to expose anything here.

Reply:
OK, removed. It was an attempt to share similar code between `PipeReader` and `StreamPipeReader`. We will see later if needed and possible.